config/create_help.txt

```diff
@@ -31,6 +31,8 @@ Import Volume Options:
  -o importVol=x                 x is the name of 3PAR volume or snapshot which needs to be imported. Volume or snapshot which needs to be imported
                                 should not be attached to any of the host.
 
+Backend Volume Options:
+ -o backend=x                   x is the backend name on which the operation to be performed.
```

Comment:
This line has to be added under create volume options field as well

Reply:
@sandesh-desai 
please remove 'Backend Volume Options'
simply keep -o backend=x and description under Import volume options